D1673/blas_interface.md

```diff
@@ -698,6 +821,50 @@ a special case of tensor algebra, users of linear algebra have
 different interface expectations than users of tensor algebra.  Thus,
 it makes sense to have two separate interfaces.
 
+### Explicit support for asynchronous return of scalar values
+
+After we presented revision 2 of this paper,
+LEWG asked us to consider support for discrete graphics processing units (GPUs).
+GPUs have two features of interest here.
+First, they might have memory that is not accessible from ordinary C++ code,
+but could be accessed in a standard algorithm
+(or one of our proposed algorithms)
+with the right implementation-specific `ExecutionPolicy`.
+(For instance, a policy could say "run this algorithm on the GPU.")
+Second, they might execute those algorithms asynchronously.
+That is, they might write to output arguments at some later time
+after the algorithm invocation returns.
+This would imply different interfaces in some cases.
+For instance, a hypothetical asynchronous vector 2-norm
+might write its scalar result via a pointer to GPU memory,
+instead of returning the result "on the CPU."
+
+Nothing in principle prevents `basic_mdspan` from viewing memory
+that is inaccessible from ordinary C++ code.
+This is a major feature of the `Kokkos::View` class
+from the [Kokkos library](https://github.com/kokkos/kokkos),
+and `Kokkos::View` directly inspired `basic_mdspan`.
+The C++ Standard does not currently define how such memory behaves,
+but implementations could define its behavior and make it work with `basic_mdspan`.
```

Comment:
Maybe add something like: For example one could envision in the future accessors which are memory space specific (a pretty direct analogy to allocators) which define which type of execution agents can access them.